### PR TITLE
Remove existing test and implement new one

### PR DESCRIPTION
**Pull request by buck06191**
## What does this change?
We remove the existing embeds test that split the audience 50/50 and replace it with one that rolls the variant out to 100% of the audience.
This is to confirm the impact of the new variant following a number of bugs in the 50/50 test.

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## What is the value of this and can you measure success?
Tracking in place for the AB test should cover this.

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/newsletter-embed-test.js [hunk 1]
```diff
@@ -66,19 +66,6 @@ export const newsletterEmbeds = {
...
             },
-        },
-        {
-            id: 'control',
-            test: () => {
-                const iframes = ((document.querySelectorAll('.email-sub__iframe')));
-                if (iframes) {
-                    iframes.forEach((ifrm) => {
-                        if (ifrm.id !== 'footer__email-form') {
-                            trackComponentInOphan(ifrm.id, 'control');
```
Comment (alexduf) [inline static/src/javascripts/projects/common/modules/experiments/tests/newsletter-embed-test.js:77]: We're still going to need tracking though aren't we? or are we going to baseline it?

Comment (buck06191) [reply static/src/javascripts/projects/common/modules/experiments/tests/newsletter-embed-test.js:77]: The intention is to not implement the control at all so I don't _think_ we need tracking here? I might be wrong but I think we're just going to compare against pre-test and see if we see a change

Comment (alexduf) [reply static/src/javascripts/projects/common/modules/experiments/tests/newsletter-embed-test.js:77]: I'm confused then, isn't there a missing piece that would force the new design to be displayed once the JS loads on the page?

Comment (buck06191) [reply static/src/javascripts/projects/common/modules/experiments/tests/newsletter-embed-test.js:77]: I think that should just be the fact that there's only one variant available in the test. If we roll out the test to 100% of the audience and only have one variant, everyone should be put in the variant group and the test logic should run as normal. This should mean that anyone on desktop/browser sees the new embeds but apps aren't affected as they fall back to the default of the existing embed

Comment (alexduf) [reply static/src/javascripts/projects/common/modules/experiments/tests/newsletter-embed-test.js:77]: ha off course! I got thrown off by the diff, but expanding the diff there's still the variant above, hence the confusion 👍

**Review by alexduf — APPROVED** [review]

**Review by jfsoul — APPROVED** [review]

**Pull request merged**